nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Updater/SdkPackageUpdater.cs

```diff
@@ -398,12 +469,18 @@ private static UpdateResult TryUpdateDependencyVersion(ImmutableArray<ProjectBui
                 : foundUnsupported
                     ? UpdateResult.NotSupported
                     : UpdateResult.NotFound;
+
+        static XmlElementSyntax WithConcreateContent(XmlElementSyntax element, string content)
```

Comment:
Why is this needed?

Reply:
Originally I was thinking it would get used multiple times, but since it only got used once I've folded it into it's single use location in my local changes (coming soon!)